app/queries/get_linked.rb

```diff
@@ -23,7 +23,10 @@ def call
   private
 
     def validate_presence_of_item!
-      return if DraftContentItem.where(content_id: target_content_id).any?
+      return if (
```

Comment:
I think this is faster:

```ruby
DraftContentItem.exists?(content_id: target_content_id) || LiveContentItem.exists?(content_id: target_content_id)
```

http://apidock.com/rails/ActiveRecord/Base/exists%3F/class